apps/website/src/server/db/show-and-tell.ts

```diff
@@ -220,6 +220,11 @@ export async function getPostById(
     include: {
       ...withAttachments.include,
       user: true,
+      modComments: {
+        include: {
+          user: true,
+        },
+      },
```

Comment:
This makes it easy to leak the internal mod comments and user data for the mods (including their email). So ideally we should have separate functions for the admin features and I'd probably want this to only explicitly select the username and not the other user fields.